docs/api/events/js_kanban_setedit_event.md

```diff
@@ -13,14 +13,15 @@ description: You can learn about the set-edit event in the documentation of the
 ### Usage
 
 ~~~jsx {}
-"set-edit": ({ cardId: string | number } | null) => void;
+"set-edit": ({ cardId: string | number, eventSource: "select-card" } | null) => void;
```

Comment:
eventSource?: "select-card" - the parameter is optional. sometimes it's not there, `undefined`